eq-publisher/src/getQuestionnaire/queries.js

```diff
@@ -234,6 +234,7 @@ exports.getQuestionnaire = `
         title
         introductionTitle
         introductionContent
+        pageDescription
```

Comment:
```suggestion
```
This line can be removed  - this file is only used for v2 Publisher, which is no longer maintained